ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/task/AbstractRetryingSwitchingPeerTask.java

```diff
@@ -137,16 +137,15 @@ private void refreshPeers() {
     // or the least useful
 
     if (peers.peerCount() >= peers.getMaxPeers()) {
-      failedPeers.stream()
-          .filter(peer -> !peer.isDisconnected())
-          .findAny()
-          .or(() -> peers.streamAvailablePeers().min(peers.getBestChainComparator()))
+      failedPeers.stream().filter(peer -> !peer.isDisconnected()).findAny().stream()
+          .min(EthPeers.MOST_USEFUL_PEER)
+          .or(() -> peers.streamAvailablePeers().min(EthPeers.MOST_USEFUL_PEER))
           .ifPresent(
               peer -> {
                 LOG.atDebug()
                     .setMessage(
                         "Refresh peers disconnecting peer {}... Waiting for better peers. Current {} of max {}")
```

Comment:
```suggestion
                        "Refresh peers disconnecting peer {} Waiting for better peers. Current {} of max {}")
```

Reply:
might be easier to read in the logs with Current{} of max{} at the beginning, if you're using the entire peer toString